assets/js/util.js

```diff
@@ -39,6 +41,17 @@ if (typeof Object.assign != 'function') {
 }
 /* eslint-enable */
 
+function debounce(func, delay) {
+    var clock = null;
```

Comment:
Add something like `delay = delay || 250` and we can go with that default delay.